src/panels/lovelace/special-rows/hui-buttons-row.ts

```diff
@@ -0,0 +1,19 @@
+import { customElement } from "lit-element";
+
+import { HuiButtonsBase } from "../components/hui-buttons-base";
+import { LovelaceRow, ButtonsRowConfig } from "../entity-rows/types";
+import { processConfigEntities } from "../common/process-config-entities";
+
+@customElement("hui-buttons-row")
```

Comment:
Will the buttons row need extra padding ?

Reply:
No, I think it looked fine. I'll add a screenshot in a bit